src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -116,15 +116,15 @@ public class RedisquesHttpRequestHandler implements Handler<HttpServerRequest> {
     private final QueueStatisticsCollector queueStatisticsCollector;
 
     public static void init(Vertx vertx, RedisquesConfiguration modConfig, QueueStatisticsCollector queueStatisticsCollector) {
-        log.info("Enable http request handler: " + modConfig.getHttpRequestHandlerEnabled());
+        log.info("Enable http request handler: {}", modConfig.getHttpRequestHandlerEnabled());
```

Comment:
Maybe we should say here

Http request handler enabled: {}

Reply:
It is not yet enabled. It is about to get enabled :)

Reply:
We just seem to output the state information. If we want to say what we are about to do then I propose to write 

Enabl*ing*

to indicate this.